bibtex-actions.el

```diff
@@ -92,6 +92,12 @@
      candidate (1- (frame-width)))) ; allow this to be configurable?
     (cdr (assoc "=key=" candidate)))))
 
+(defun bibtex-actions--affixation ()
+  (cl-loop
+   for candidate in (bibtex-actions--get-candidates)
+   collect
+   (candidate "ABC" "XYZ")))
```

Comment:
I'm trying to get started on this, but am stuck at the what this affixation function should be returning.